includes/definitions/edgeswitch.yaml

```diff
@@ -11,21 +11,21 @@ over:
 discovery:
     -
         sysObjectID:
+            - .1.3.6.1.2.1.1.1.0
             - .1.3.6.1.4.1.4413
             - .1.3.6.1.4.1.41112
             - .1.3.6.1.4.1.10002
             - .1.3.6.1.4.1.8072
             - .1.3.6.1.4.1
+            - .1.3.6.1.4.1.8072.3.2.10
+            - .1.3.6.1.4.1.27282.3.2.10            
         sysDescr_regex:
             - '/^EdgeSwitch/'
             - '/^EdgePoint/'
             - '/^USW[ -]/'
             - '/^UBNT US/'
             - '/^US[ -]/'
-    -
```

Comment:
This change is not correct either. The move of the sysObjectIDs that were below up to the top changes the behaviour. 
Adding `- .1.3.6.1.2.1.1.1.0` is the only thing you need to do in this PR.